Tests/Shared.Specs/StringAssertionSpecs.cs

```diff
@@ -2043,6 +2043,138 @@ public void When_the_actual_string_equivalent_to_the_expected_but_with_trailing_
 
         #endregion
 
+        #region Not Be Equivalent To
+
+        [Fact]
+        public void When_strings_are_the_same_while_ignoring_case_it_should_throw()
+        {
+            //-----------------------------------------------------------------------------------------------------------
+            // Arrange
+            //-----------------------------------------------------------------------------------------------------------
+            string actual = "ABC";
+            string unexpected = "abc";
+
+            //-----------------------------------------------------------------------------------------------------------
+            // Act
+            //-----------------------------------------------------------------------------------------------------------
+            Action action = () => actual.Should().NotBeEquivalentTo(unexpected);
+
+            //-----------------------------------------------------------------------------------------------------------
+            // Assert
+            //-----------------------------------------------------------------------------------------------------------
+            action.Should().Throw<XunitException>()
+#if NETCOREAPP1_1
+                .WithMessage("Expected string not to be equivalent to \"abc\", but they are.");
+#else
+                .WithMessage("Expected actual not to be equivalent to \"abc\", but they are.");
+#endif
+        }
+
+        [Fact]
+        public void When_strings_differ_other_than_by_case_it_should_not_throw()
+        {
+            //-----------------------------------------------------------------------------------------------------------
+            // Act
+            //-----------------------------------------------------------------------------------------------------------
+            Action act = () => "ADC".Should().NotBeEquivalentTo("abc", "we will test {0} + {1}", 1, 2);
```

Comment:
When calling `NotThrow` you can omit `because` and `becauseArgs`.
(I've read that you love to delete code)

Reply:
Fair point. Since I copied all the StringAssertions::BeEquivalentTo test and inverted them, things got left over. 👍